src/Symfony/Component/DependencyInjection/Argument/BoundArgument.php

```diff
@@ -16,35 +16,48 @@
  */
 final class BoundArgument implements ArgumentInterface
 {
+    const SERVICE_BIND = 0;
+    const DEFAULTS_BIND = 1;
+    const INSTANCEOF_BIND = 2;
+
+    public static $messages = [
```

Comment:
I'd suggest removing this variable, it's not needed.

Reply:
Done